docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+services:
+  mysql:
+    image: mysql:latest
+    container_name: mysql_container
+    environment:
+      MYSQL_ROOT_PASSWORD: 1234
```

Comment:
Using hardcoded passwords like '1234' for `MYSQL_ROOT_PASSWORD` is a security risk. Consider using environment variables or Docker secrets to manage sensitive information securely.